docs/Axis-Title-menu-layout.md

```diff
@@ -0,0 +1,81 @@
+
+Split axis could be done either before or after adding datasets to the axis.
+
+![2a-split-axis](https://github.com/user-attachments/assets/26e7b4a8-b6e3-42f1-a8c3-c9110efe06ac)
+
+### Zooming the axis
+
+Upon zooming in the axis (Click and drag) the representation of individual features transitions from a histogram to a more distinct fingerprint view.
+
+![2b-split-axis-zoom-in](https://github.com/user-attachments/assets/9a4f931e-5cda-450f-92c6-75f93d4ba1e7)
+
+### Viewing the features 
+
+=== "Adding Feature"
+
+    Selecting an individual fingerprint band allows the user to view the feature's location, represented by an arrow, and   selecting the arrow displays the feature name. 
+    
+    ![2c-split-axis-add-feature](https://github.com/user-attachments/assets/30f19a38-cc6c-4792-9f6f-bf568122ef2a)
+     
+=== "Hiding Feature"
+    
+    Clicking the feature arrow hides the feature name, and selecting the fingerprint band again hides the feature entirely.
+    
+    ![2d-split-axis-remove-feature](https://github.com/user-attachments/assets/2cc78f74-0d63-4d5c-890a-b344e8c9456c)
+    
+!!!note
+  Deselect any displayed feature to ensure it is not automatically displayed the next time the same dataset is opened.
+
+## Flip Axis
+
+To flip or invert the axis, open the Axis Title menu and select the icon with up and down pointing arrows. For instance, 
+
+![3-flip-axis](https://github.com/user-attachments/assets/d617c796-1ab5-4382-a063-66d9861ca70f)
+
+![basic-function_axis-flip-orientation](https://github.com/user-attachments/assets/2bd66224-a23e-47e0-8c18-5aeeb9ec6e3a)
+
+## Removing Datasets from visualization
+
+=== "Specific Dataset"
+
+    Open the Axis Title menu and click the cross icon next to a specific dataset to remove it from the displayed view.
+    
+    ![5a  remove selected dataset](https://github.com/user-attachments/assets/ec022dd1-6f93-4448-b89c-bb2c86b46f54)
+    
+       
+=== "All Datasets"
+
+    To remove all datasets from view, open the Axis Title menu and click the cross icon located above the axis name.
+    
+    ![5b  Removing all datasets](https://github.com/user-attachments/assets/038e5990-b8cd-4e99-bbf3-32903739f647)
+    
+## Rename Axis title
+
+To rename the axis title, open the Axis Title menu and click the pen icon to access the text bar. Enter a new axis title that aligns with your objective.
+
+![4 rename](https://github.com/user-attachments/assets/62495c11-99c1-4f73-b101-c3e2a47e258a)
+
+## Display and hide features of dataset
+
+Open the Axis Title menu and click the eye icon to toggle the display of a particular dataset, showing or hiding it from view.
+
+=== "Display feature"
+
+    ![6a-hide-features](https://github.com/user-attachments/assets/e0d58454-09b0-4466-b51b-db4d98df0c65)
+
+=== "Hide feature"
+
+    ![6b-display-features](https://github.com/user-attachments/assets/a123b35a-f335-4cbb-832d-6b119734e6e2)
+
+## Selected features vs total features of a dataset
```

Comment:
Please change this to "Requested features vs total features of a dataset"

Reply:
Updated